### PR TITLE
Make model class customisable

### PR DESCRIPTION
**Pull request by jack89roberts**
and lint. Builds on Ryan's confederation-priors branch

### CODE REVIEW

File: wcpredictor/data/confederations.csv [hunk 1]
```diff
@@ -210,3 +210,108 @@ Hungary,UEFA
 Azerbaijan,UEFA
 Netherlands,UEFA
 San Marino,UEFA
+Andalusia,Unaffiliated
```
Comment (nbarlowATI) [inline wcpredictor/data/confederations.csv:213]: Do we need these unaffiliated teams?    Are we not just excluding them anyway in `get_results_data`?

File: wcpredictor/data/fifa_game_rankings.csv [hunk 2]
```diff
@@ -15,6 +15,8 @@ Poland,80,75,74,77
 Morocco,80,74,78,77
 Mexico,78,77,76,77
 USA,74,76,74,75
+Columbia,75,75,75,75
```
Comment (nbarlowATI) [inline wcpredictor/data/fifa_game_rankings.csv:18]: Should this be "Colombia" ?

**Review by nbarlowATI — APPROVED** [review]
Looks good - I guess we should have a linted version in main sooner rather than later anyway :-)   
Not quite sure if/how the extra non-affiliated teams, and all those extra competitions, are used, but they won't do any harm anyway.
Maybe "Columbia" -> "Colombia" fix before merging?

File: wcpredictor/data/confederations.csv (hunk 1 above)
Comment (jack89roberts) [reply wcpredictor/data/confederations.csv:213]: I added them to give the option of running on the full dataset (though it likely doesn't make much difference).

File: wcpredictor/data/fifa_game_rankings.csv (hunk 2 above)
Comment (jack89roberts) [reply wcpredictor/data/fifa_game_rankings.csv]: fixed

**Pull request merged**